src/rdo.rs

```diff
@@ -292,23 +292,22 @@ pub fn rdo_mode_decision(
     cw.find_mvrefs(bo, LAST_FRAME, &mut mv_stack, bsize, false);
 
   for &luma_mode in &mode_set {
-    assert!(fi.frame_type == FrameType::INTER || luma_mode.is_intra());
+    let luma_mode_is_intra = luma_mode.is_intra();
```

Comment:
Not sure there is any benefit in doing this. Do you have one in mind?

Reply:
Yes, they are not for speed up, but simply meant to avoid calling .is_intra() functions, which gives the same info.